instances/k8smaster/main.tf

```diff
@@ -12,12 +12,14 @@ resource "oci_core_instance" "TFInstanceK8sMaster" {
   shape               = "${var.shape}"
   subnet_id           = "${var.subnet_id}"
 
-  metadata {
+  extended_metadata {
     roles               = "masters"
     ssh_authorized_keys = "${var.ssh_public_key_openssh}"
     user_data           = "${data.template_cloudinit_config.master.rendered}"
+    tags = "group:k8s-master"
```

Comment:
this looks like an ansible tag.

Reply:
I purposely added these tags to illustrate using the new extended_metadata fields. I could remove the `group` portion to make it more generic.

Reply:
ok.  looks good